### ISSUE TITLE
Invoke hub's method which throw ArgumentException, client gets HubException: An unexpected error occurred

### ISSUE THREAD

**Issue by Jun2014** [Azure/azure-signalr#488]
We have a method on hub:
```
        public string GetText(string apiKey)
        {
                throw new ArgumentException("APIKEY_INVALID");
        }
```
Now client invokes this method, and get back a generic exception which doesn't help client to diagnose this error.
```
Microsoft.AspNetCore.SignalR.HubException: An unexpected error occurred invoking 'GetText' on the server.
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.SignalR.Client.HubConnection.<InvokeCoreAsyncCore>d__51.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
```

We're using Microsoft.AspNetCore.SignalR 1.1.0 on server, and Microsoft.AspNetCore.SignalR.Client 1.1.0 on client.

**Comment by BrennanConroy** [comment]
https://docs.microsoft.com/en-us/aspnet/core/signalr/hubs?view=aspnetcore-2.2#handle-errors

TLDR: Sending the exception info back to the client is considered insecure as you could accidentally leak data. If you want to see the exception on the client then either throw a `HubException` or set `EnableDetailedErrors` on the server.

**Comment by Jun2014** [comment]
This works, thanks.
